activesupport/test/broadcast_logger_test.rb

```diff
@@ -3,75 +3,147 @@
 module ActiveSupport
   class BroadcastLoggerTest < TestCase
     attr_reader :logger, :log1, :log2
-    def setup
+
+    setup do
       @log1 = FakeLogger.new
       @log2 = FakeLogger.new
       @log1.extend Logger.broadcast @log2
       @logger = @log1
     end
 
-    def test_debug
-      logger.debug "foo"
-      assert_equal 'foo', log1.adds.first[2]
-      assert_equal 'foo', log2.adds.first[2]
+    Logger::Severity.constants.each do |level_name|
```

Comment:
If this is too clever I can remove it. It will just mean copying the same 5 line test 6 times for each log level.